awx_collection/plugins/modules/tower_inventory_source.py

```diff
@@ -195,6 +201,12 @@ def main():
     source_project = module.params.get('source_project')
     state = module.params.get('state')
 
+    new_fields = {}
+    organization_id = None
+    organization = module.params.get('organization')
+    if organization:
+        organization_id = module.get_one_by_name_or_id('organizations', organization)
+
     # Attempt to look up inventory source based on the provided name and inventory ID
     inventory_id = module.resolve_name_to_id('inventories', inventory)
```

Comment:
Need to actually make use of `organization_id` in the query for this to make sense.

Reply:
In other cases this was done by using `get_one`, but in this case it seems appropriate for a new helper method that will fail if multiples are returned like `get_exactly_one`